genestack_client/data_importer.py

```diff
@@ -122,6 +122,19 @@ def __invoke_loader(self, parent, importer_type, metainfo):
         self.__process_links(metainfo)
         return self.importer.invoke('createFile', parent, importer_type, metainfo)['accession']
 
+    def __add_to_metainfo(self, metainfo, key, value, setter, required=False):
```

Comment:
I've written a couple of lines before realizing that I was wrong about some details behind the function =)
So, there's a need for doctsring!